iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/serviceclient/ServiceClientTests.java

```diff
@@ -125,14 +126,14 @@ public static void stopProxy()
     @StandardTierHubOnlyTest
     public void cloudToDeviceTelemetry() throws Exception
     {
-        cloudToDeviceTelemetry(false, true, false, false);
+        cloudToDeviceTelemetry(false, true, false, false, false);
```

Comment:
this is where I wish github had intellisense! 😉

Reply:
Tim just needs to create an over complex truth table so we can see this at a glance :)